services/db/recovery.go

```diff
@@ -149,6 +150,24 @@ func alertOps(mode string) {
 	}
 }
 
+// sendRecoveryModeStat sends the recovery mode metric every 1 minute
+func sendRecoveryModeStat() {
+	for {
+		time.Sleep(1 * time.Minute)
```

Comment:
Can you send it every 10 seconds instead because that's our metric collection interval usually. 

Do you think we should make this configurable?

Reply:
Changed it to 10 seconds. Don't think we need to make it configurable. It will be easier for us to define alerts on it if we make this fixed. WDYT?

Reply:
Sure. It's fine.